tests/pipelines/eks/awscli-eks-cl2-load.yaml

```diff
@@ -134,6 +143,9 @@ spec:
     taskRef:
       kind: Task
       name:  awscli-eks-nodegroup-create
+    workspaces:
+    - name: config    
+      workspace: config
   - name: generate
```

Comment:
can you take off endpoints param from this file too for tasks that it's not necessary, like you did for addons-pipeline ?

Reply:
done.